app/models/broadcast.rb

```diff
@@ -3,6 +3,7 @@ class Broadcast < ActiveRecord::Base
 
   has_many(:broadcast_addressings, :inverse_of => :broadcast, :dependent => :destroy)
   has_many(:recipients, :through => :broadcast_addressings, :source => :user)
+  has_many(:sms_messages, :dependent => :destroy, :class_name => 'Sms::Message')
```

Comment:
Perhaps best. These days I try to restrict associations to those needed since each one adds about 9 billion methods to the class' public interface.